src/components/views/Leaderboard.tsx

```diff
@@ -117,6 +124,10 @@ export function Leaderboard(props: Props) {
             setValue={setSelectedFaction}
             class="flex-auto sm:flex-none"
           />
+          <div class={classes("btn-group")}>
```

Comment:
You can use `styles.button.set` here. The `classes()` function is redundant as it only joins together multiple strings or conditionals to form once CSS class.